erdpy/delegation/staking_provider.py

```diff
@@ -0,0 +1,141 @@
+import binascii
+from typing import Any
+from os import path
+
+from erdpy.accounts import Account
+from erdpy.config import MetaChainSystemSCsCost
+from erdpy.utils import read_json_file_validators, parse_keys
+from erdpy.validators.core import estimate_system_sc_call
+from erdpy.wallet.pem import parse_validator_pem
+from erdpy.wallet.signing import sign_message_with_bls_key
+
+DELEGATION_MANAGER_SC_ADDRESS = "erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqylllslmq6y6"
+
+
+def prepare_args_for_create_new_staking_contract(args: Any):
+    args.data = 'createNewDelegationContract'
+    args.data += '@' + string_to_hex_bytes(str(args.total_delegation_cap))
+    args.data += '@' + string_to_hex_bytes(str(args.service_fee))
+
+    args.receiver = DELEGATION_MANAGER_SC_ADDRESS
+
+    if args.estimate_gas:
+        args.gas_limit = estimate_system_sc_call(args, MetaChainSystemSCsCost.DELEGATION_MGR_OPS, 3)
+
+
+def string_to_hex_bytes(number_str: str) -> str:
+    num_of_bytes = 1
```

Comment:
Similar logic already exists in `_prepare_argument`. Sometime (not now), we have to refactor and extract the commonality.

Reply:
And better: add specialized classes that convert stuff into other stuff and then should place them together in module like "converters"

Reply:
done. create a class Converter in package conv